Tracing: Release trace to logs feature (#29443)
grafana/grafana: docs/sources/datasources/jaeger.md

Grafana ships with built-in support for Jaeger, which provides open source, end-to-end distributed tracing.
Just add it as a data source and you are ready to query your traces in [Explore]({{< relref "../explore/index.md" >}}).

## Add data source

> To access Jaeger settings, click the **Configuration** (gear) icon, then click **Data Sources**, and then click **Jaeger**.

Reviewer: ```suggestion
To access Jaeger settings, click the **Configuration** (gear) icon, then click **Data Sources** > **Jaeger**.
```

| Name          | Description                                                                                                                           |
| ------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
| `Name`        | The data source name. This is how you refer to the data source in panels, queries, and Explore.                                       |
| `Default`     | Data source will be pre-selected for new panels.                                                                                      |
| `URL`         | The URL of the Jaeger instance, for example, `http://localhost:16686`                                                                 |

> | `Access`      | Server (default) = URL needs to be accessible from the Grafana backend/server, Browser = URL needs to be accessible from the browser. |

Reviewer: ```suggestion
| `Access`      | Server (default) = URL needs to be accessible from the Grafana backend/server. Browser = URL needs to be accessible from the browser. |
```

| `Basic Auth`  | Enable basic authentication to the Jaeger data source.                                                                                |
| `User`        | User name for basic authentication.                                                                                                   |
| `Password`    | Password for basic authentication.                                                                                                    |
| `Data source` | Data source for trace to logs feature.                                                                                                |

> | `Tags`        | The tags that will be used in the Loki query. Default is `'cluster', 'hostname', 'namespace', 'pod'`                                  |

Reviewer: ```suggestion
| `Tags`        | The tags that will be used in the Loki query. Default is `'cluster', 'hostname', 'namespace', 'pod'`.                                  |
```

Just add it as a data source and you are ready to query your traces in [Explore]({{< relref "../explore/index.md" >}}).

> ## Adding the data source

Reviewer: ```suggestion
## Add data source
```

To access Jaeger settings, click the **Configuration** (gear) icon, then click **Data Sources**, and then click **Jaeger**.

| Name          | Description                                                                                                                           |
| ------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
| `Name`        | The data source name. This is how you refer to the data source in panels, queries, and Explore.                                       |

> | `Default`     | Default data source means that it will be pre-selected for new panels.                                                                |

Reviewer: ```suggestion
| `Default`     | Data source will be pre-selected for new panels.                                                                |
```

> | `URL`         | The URL of the Jaeger instance, e.g., `http://localhost:16686`                                                                        |

Reviewer: ```suggestion
| `URL`         | The URL of the Jaeger instance, for example, `http://localhost:16686`                                                                        |
```
